functionary/ui/templates/core/scheduledtask_detail.html

```diff
@@ -59,16 +59,18 @@ <h3 class="fw-semibold">
                 <a href="{% url 'ui:function-detail' scheduledtask.function.id %}">{{ scheduledtask.function.render_name }}</a>
             </p>
         </div>
-        {% if scheduledtask.parameters %}
-            <div>
-                <h3 class="fw-semibold">
-                    <i class="fa fa-list fa-sm fa-fw me-1"></i>Parameters
-                </h3>
-                <div class="my-2">
+        <div>
+            <h3 class="fw-semibold">
+                <i class="fa fa-list fa-sm fa-fw me-1"></i>Parameters
+            </h3>
+            <div class="my-2">
+                {% if scheduledtask.parameters %}
                     <ol class="json-container wrap-json" id="json-result" />
-                </div>
+                {% else %}
+                    <p class="font-monospace">No parameters defined</p>
```

Comment:
The wording should be consistent between this and task_detail.html.  Probably just "No parameters" is fine.

Also, everything else has `ms-4`, so the text here feels misaligned.